src/interfaces/slack_bot/src/constants.ts

```diff
@@ -0,0 +1,151 @@
+import { Action } from './utils/actions';
+
+export const COMMAND_SUFFIX = process.env.COMMAND_SUFFIX || '';
+export const COHERE_API_TIMEOUT_SECONDS = 60 * 5; // 5 minutes
+
+// todo: change to follow the format of the other envs above
+export const DEFAULT_CHAT_TEMPERATURE = 0.3;
+export const DEFAULT_MODELS = {
+  CHAT: 'command-nightly',
```

Comment:
This shouldn't be nightly - can we not default here?

Reply:
I have changed it to use `command-r` as the default. It currently constructs the chat params like this:

```
  const params: CohereChatRequest = {
    message: messageWithoutUsername,
    model: model ?? DEFAULT_MODELS.CHAT,
    conversation_id: conversationId,
    prompt_truncation: 'AUTO_PRESERVE_ORDER',
  };
```

Where model is the value that a user has set with the `/set-model` command in slack. I could rework this to remove passing `model` by default, and instead just add model to the params if it has a value.

Reply:
I have gone ahead and made this change. It will no longer pass a default model to the toolkit API, and instead will only pass the `model` param when it has been explicitly set by the user with the `/set-model` command in Slack.